app/views/business_mailer/new_developer_profile.text.erb

```diff
@@ -0,0 +1,15 @@
+New developers at Railsdevs.com!
+
+<%= @business.name %>,
+
+Here's your digest of new developers available for hire at Railsdevs.com:
+
+<%= "-" * 20 %>
+
+<% @developers.each do |dev| %>
+<%= dev.name %>: <%= dev.hero %>
```

Comment:
We can still link to the developer.

```suggestion
<%= developer_url(developer) %>
<%= dev.name %>: <%= dev.hero %>
```

Reply:
See below. Plain text emails do not allow hyperlinking.